src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ProcessResponse.liquid

```diff
@@ -6,45 +6,32 @@ var fileResponse_ = new FileResponse((int)response_.StatusCode, headers_, respon
 client_ = null; response_ = null; // response and client are disposed by FileResponse
 return fileResponse_;
 {%         else -%}
-var responseData_ = response_.Content == null ? null : await response_.Content.ReadAsStringAsync().ConfigureAwait(false); 
-throw new {{ ExceptionClass }}("{{ response.ExceptionDescription }}", (int)response_.StatusCode, responseData_, headers_, null);
+ObjectResponseResult<{{ response.Type }}> objectResponse_ = await this.ReadObjectResponseAsync<{{ response.Type }}>(response_, headers_).ConfigureAwait(false);
```

Comment:
Do not use `this.`

Reply:
I assume that's to allow the use of `static` implementations?